src/fu-util.c

```diff
@@ -2564,14 +2611,14 @@ fu_util_update(FuUtilPrivate *priv, gchar **values, GError **error)
 						 &error_local);
 		if (rels == NULL) {
 			if (!latest_header) {
-				g_printerr(
-				    "%s\n",
+				fu_console_println_literal(
+				    priv->console,
 				    /* TRANSLATORS: message letting the user know no device upgrade
 				     * available */
 				    _("Devices with the latest available firmware version:"));
 				latest_header = TRUE;
 			}
-			g_printerr(" • %s\n", fwupd_device_get_name(dev));
+			fu_console_println(priv->console, " • %s", fwupd_device_get_name(dev));
```

Comment:
same stderr comment

Reply:
I think this one actually wants to be on stdout.